README.md

```diff
@@ -0,0 +1,51 @@
+# simple-blockchain
+
+### install
+
+1. download the file
```

Comment:
I think this document have to address how to install LevelDB. (e.g. `brew install leveldb`)